homework_1/task_01/fizzBuzz/fizzBuzz.go

```diff
@@ -0,0 +1,36 @@
+package fizzBuzz
```

Comment:
https://axilis.slack.com/archives/C020JHMJ274/p1620815297002300?thread_ts=1620815275.002100&cid=C020JHMJ274

Make sure to rename the test package as well.